Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -306,68 +321,55 @@ bool copyPropertyVector(MeshLib::Properties const& original_properties,
     return true;
 }
 
+/// Applies a function of the form f(type, name) -> bool for each of the
+/// properties names.
+/// The type argument is used to call f<decltype(type)>(name).
+/// At least one of the functions must return the 'true' value, but at most one
+/// is executed.
 template <typename Function>
-void copyPropertyVectors(std::vector<std::string> const& property_names,
-                         Function copy_property_vector)
+void applyPropertyVectors(std::vector<std::string> const& property_names,
```

Comment:
OK, It's an abstraction. But is that really needed? And maybe `applyToPropertyVectors` would be a better name.